README.md

```diff
@@ -1,3 +1,166 @@
 # Cluster Node Tuning Operator
 
-The Cluster Node Tuning Operator manages cluster node-level tuning for [OpenShift](https://openshift.io/).
+The Cluster Node Tuning Operator manages cluster node-level tuning for
+[OpenShift](https://openshift.io/).
+
+The majority of high-performance applications require some
+level of kernel tuning.  The operator provides a unified
+management interface to users of node-level sysctls and more
+flexibility to add [custom tuning](#custom-tuning-specification)
+specified by user needs.  The operator manages containerized
+[tuned](https://github.com/redhat-performance/tuned/)
+daemon for [OpenShift](https://openshift.io/) as
+a Kubernetes DaemonSet.  It ensures [custom tuning
+specification](#custom-tuning-specification) is passed to all
+[containerized tuned](https://github.com/openshift/openshift-tuned)
+daemons running in the cluster in the format they understand.
+The daemons run on all nodes in the cluster, one per node.
+
+
+## Deploying the node tuning operator
+
+The operator is deployed by applying the manifests in the operator's
+`/manifests` directory.  It tries to be useful out of the box by
```

Comment:
I would reword this sentence to say "It automatically creates a default deployment and configuration
([CR](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/))
for the tuned daemons" but then indicate what makes this a useful feature. Is it useful because it saves the user time, or adds value in another way?

Reply:
The first instance of "custom resource", I would spell out as "custom resource (CR)". Any instance afterwards can be just "CR"